source/server/getting-started/configuration/sponge-conf.rst

```diff
@@ -13,464 +13,1427 @@ Below is a list of available config settings and their comments inside the globa
-    "The client has sent too many requests within a certain amount of time".
-    Finally, if set to 0 or less, the default interval will be used.
-generate-spawn-on-load
-    If 'true', this world will generate its spawn the moment its loaded.
-infinite-water-source
-    Vanilla water source behavior - is infinite
-invalid-lookup-uuids
-    The list of uuid's that should never perform a lookup against Mojang's session server.
-    Note: If you are using SpongeForge, make sure to enter any mod fake player's UUID to this list.
-item-merge-radius
-    The defined merge radius for Item entities such that when two items are
-    within the defined radius of each other, they will attempt to merge. Usually,
-    the default radius is set to 0.5 in Vanilla, however, for performance reasons
-    2.5 is generally acceptable.
-    Note: Increasing the radius higher will likely cause performance degradation
-    with larger amount of items as they attempt to merge and search nearby
-    areas for more items. Setting to a negative value is not supported!
-keep-spawn-loaded
-    If 'true', this worlds spawn will remain loaded with no players.
-leaf-decay
-    If 'true', natural leaf decay is allowed.
-load-on-startup
-    If 'true', this world will load on startup.
-max-chunk-unloads-per-tick
-    The maximum number of queued unloaded chunks that will be unloaded in a single tick.
-    Note: With the chunk gc enabled, this setting only applies to the ticks
-    where the gc runs (controlled by 'chunk-gc-tick-interval')
-    Note: If the maximum unloads is too low, too many chunks may remain
-    loaded on the world and increases the chance for a drop in tps. (Default: 100)
-mob-spawn-range
-    Specifies the radius (in chunks) of where creatures will spawn.
-    This value is capped to the current view distance setting in server.properties
-portal-agents
-    A list of all detected portal agents used in this world.
-    In order to override, change the target world name to any other valid world.
-    Note: If world is not found, it will fallback to default.
-pvp-enabled
-    If 'true', this world will allow PVP combat.
-view-distance
-    Override world distance per world/dimension
-    The value must be greater than or equal to 3 and less than or equal to 32
-    The server-wide view distance will be used when the value is -1.
-weather-ice-and-snow
-    If 'true', natural formation of ice and snow in supported biomes will be allowed.
-weather-thunder
-    If 'true', thunderstorms will be initiated in supported biomes.
-world-enabled
-    If 'true', this world will be registered.
 
 ------------------------------------------------------------------------------------------------------------
 
-This config was generated using SpongeForge build 3442 (with Forge 2705), SpongeAPI version 7.1.0:
+This config was generated using SpongeForge recommendation 5 (SpongeForge 3554 with Forge 2768), SpongeAPI version 7.1:
```

Comment:
This does not read well and is technically incorrect (recommendation 5 is not a thing, it's the fifth recommended build that implements Sponge API 7.1). 

Simply say "This configuration file was generated using SpongeForge 7.1.5". That points to a specific build.

Reply:
I have suggested before that we could version the sponge.conf file. Whether it's worth the effort is anyone's guess, but the SF/SV build number is a good compromise.

Reply:
Done